docs/windows/latest-supported-vc-redist.md

```diff
@@ -0,0 +1,94 @@
+| ------------ | :--------------------------------------------------------------------------------------------------: | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| ARM64        | [https://aka.ms/vs/16/release/vc_redist.arm64.exe](https://aka.ms/vs/16/release/vc_redist.arm64.exe) |                                                                                                                                                                                               Permalink for latest supported ARM64 version |
+| X86          |   [https://aka.ms/vs/16/release/vc_redist.x86.exe](https://aka.ms/vs/16/release/vc_redist.x86.exe)   |                                                                                                                                                                                                 Permalink for latest supported x86 version |
+| X64          |   [https://aka.ms/vs/16/release/vc_redist.x64.exe](https://aka.ms/vs/16/release/vc_redist.x64.exe)   | Permalink for latest supported x64 version. To facilitate installing required Visual C++ ARM64 binaries when the X64 redistributable is installed on an ARM64 device, the X64 redistributable package contains both ARM64 and X64 binaries |
+
+### Notes
+
+- Visual Studio 2015, 2017 and 2019 share the same redistributable files. For example, the Microsoft Visual C++ 2019 redistributable will be used by C++ programs built with the Visual Studio 2015, 2017, or 2019 toolsets. However, the version of the Microsoft Visual C++ redistributable installed on the machine must be the same or higher than the version of the Visual C++ toolset used to create your application. Please see [Determining Which DLLs to Redistribute](determining-which-dlls-to-redistribute.md) for more information about which version you need.
+- **Windows XP Support**: Microsoft ended support for Windows XP on April 8, 2014. While the current version of the C++ Redistributable for Visual Studio 2015-2019 only supports Windows Vista, 7, 8.1, 10, and 11 older versions of the C++ Redistributable 2015-2019 can be installed on Windows XP. The last version of the C++ Redistributable for Visual Studio 2015-2019 that works on Windows XP is v14.24.28127.4. Download from [My.VisualStudio.com](https://my.visualstudio.com/) or directly using the following link corresponding to your architecture: [vc_redist.86.exe](https://download.visualstudio.microsoft.com/download/pr/bc2490f7-3f8e-4a5d-a61e-12b6d331713e/E59AE3E886BD4571A811FE31A47959AE5C40D87C583F786816C60440252CD7EC/VC_redist.x86.exe) or [vc_redist.x64.exe](https://download.visualstudio.microsoft.com/download/pr/3b070396-b7fb-4eee-aa8b-102a23c3e4f4/40EA2955391C9EAE3E35619C4C24B5AAF3D17AEAA6D09424EE9672AA9372AEED/VC_redist.x64.exe)
```

Comment:
Regarding "Windows Vista, 7, 8.1, 10, and 11 older versions of the C++ Redistributable 2015-2019", that reads "11 older versions of the C++..."   Put a comma after 11 to fix that.